sapp/pipeline/mariana_trench_parser.py

```diff
@@ -24,12 +24,6 @@
 from ..analysis_output import AnalysisOutput, Metadata
 from .base_parser import BaseParser
 
-try:
-    # type: ignore
-    from ...facebook.lib import configuration
-except Exception:
-    from .. import configuration
-
```

Comment:
Interesting that `flake8` seems to think these imports are unused, since [`from .. import configuration` is a valid path](https://github.com/facebook/sapp/blob/master/sapp/configuration.py). Maybe because the imports are wrapped in a try-except block?

Anyway, we need to have the linter ignore these lines, because if we delete these imports, we will break `sapp analyze` pipeline for internally and externally for one of our static analysis tools

Reply:
It was removed because it was unused. I will add it back if it's needed :)

Reply:
Oh, my bad! I'm not sure how I missed this earlier, but it doesn't look like the file is used at all anymore

So we can leave the deletion of the entire try-except block here and can also delete [sapp/configuration.py](https://github.com/facebook/sapp/blob/master/sapp/configuration.py) as well. Sorry about missing this on my first pass :(

Reply:
@0xedward it's fine, I will just revert and check and delete the file as well :)